### PR TITLE
Add the SI prefixes to the index of library names

### PR DESCRIPTION
**Pull request by marcmutz**

### CODE REVIEW

**Review by jwakely — APPROVED** [review]

File: source/meta.tex [hunk 1]
```diff
@@ -2626,6 +2646,10 @@
 \rSec2[ratio.si]{SI types for \tcode{ratio}}
 
 \pnum
+\indexlibraryglobal{yocto}%
+\indexlibraryglobal{zepto}%
+\indexlibraryglobal{zetto}%
```
Comment (JohelEGP) [inline source/meta.tex:2651]:
```suggestion
\indexlibraryglobal{zetta}%
```

**Review by JohelEGP — CHANGES_REQUESTED** [review]

**Review by JohelEGP — APPROVED** [review]

**Comment by tkoeppe** [comment]
I don't think this is editorial?!

**Comment by marcmutz** [comment]
> How about using `\libglobal`?

Good idea. I have no idea what it's doing, but I see how it's used for ratio_equal_v etc, and I like how it's inline that way. Will update.

**Comment by JohelEGP** [comment]
It's not mentioned at https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#indexes, but it links to the page of the declaration (or declaration itself on eel.is), rather than the beginning of the synopsis. It also avoids spelling errors.

**Review by JohelEGP — APPROVED** [review]

**Review by jensmaurer — APPROVED** [review]

**Pull request merged**